active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,356 @@
+        * An example is if the user is interacting with the number panel mode and the focus is on the page 2 button, when the user hits page down, the focus will move to the page 12 button (if page 12 exists) or the last page button. 
+* Home/escape keys should move focus from an individual item back to the entire control.
+* (Combo box mode only) If the end user is focused on the combo box, the user can hit enter to expand the drop down menu.
+* (Combo box mode only) If the end user is focused on the combo box and the drop down is expanded, the home/escape key should collapse the drop down menu and have focus remain on the combo box. 
+* (Combo box mode only) Alt + down shortcut should expand the drop down menu 
+* (Combo box mode only) Alt + up shortcut should collapse the drop down menu
+* (Number box mode only) Up and down arrows should increase or decrease the value in the number box by 1. 
+
+### Narrator
+* Should be added as a navigation landmark. 
+* When the focus is on the control, narrator will announce _____. 
+* When the focus is on the first, last, previous, or next button narrator will announce "first page", "last page", "next page", or "previous page". 
+        * If the buttons have text properties set by the developer, narrator will announce that text instead of the default announcement. 
+* When the focus is on a number button, narrator will announce " page x". 
+* Combo box and number box modes will use the default narrator announcements for combo box and number box. 
+
+## Data and Intellegence Metrics 
+
+Adoption and validation of the control in the community
+* Measurement: Use telemetry to determine how many first and third party developers are using the control in preview. 
+* Measurement: Count of posts providing feedback to us through Github and Discord. 
+
+Questions that can be answered through telemetry metrics to help focus future features and improvements: 
+
+Which display mode is the most commonly used? 
+* Measurement: Count of how many applications include the pager control type broken out by display mode.
+
+Which layout view is this control being used with the most?  
+ * Help drive improvements for keyboarding and to detemine which layout views we should focus on adding pager control to first. I'm not sure if this is something that can be measured. 
+
+How often are developers using this control with an indefinite number of pages?
+* Measurement: Count of how many applications include the pager control type and have the NumberOfPages property empty. 
+
+
+## Open Questions 
+
+* I need help understanding how to choose what automation control type could be used for a control like this. Can it be a combination of control types?
+* Can the default behavior for combo box and number box be added to pager control? Does anything special need to happen in order to get that functionality?
+* I'm not sure what narrator should announce when focus is on the control. 
```

Comment:
I think it should announce the UIA name given by the developer (similiar to NavigationView I think). That way, if there is a meaningful name, the developer can provide it.

Reply:
That is a really good suggestion! I was debating between saying "Pager" or something a little more clever as the default but I like your suggestion about the developer giving it a name.